btrack/include/tracker.h

```diff
@@ -20,7 +20,7 @@
 #include <algorithm>
 #include <cmath>
 #include <ctime>
-#include <filesystem>
+// #include <experimental/filesystem>
```

Comment:
Here and elsewhere should we just remove rather than comment out?

Reply:
I think I'd like to leave this in if possible, since we might want to reinstate it once we can move back to C++17 (or greater). Probably fine to delete though, it's not an essential function - mostly used for debugging.